Microsoft.Orleans.OrleansCodeGenerator.Build dotnet core 2.2 support

Hello,

Today I've tried to update my Orleans project to dotnet core 2.2 and faced the following issue. Seems that Microsoft.Orleans.OrleansCodeGenerator.Build package (https://www.nuget.org/packages/Microsoft.Orleans.OrleansCodeGenerator.Build/2.2.0-rc1) supports dotnet core 2.1 https://github.com/dotnet/orleans/blob/master/src/Orleans.CodeGeneration.Build/build/Microsoft.Orleans.OrleansCodeGenerator.Build.targets and in latest RC-1 for orleans 2.2.0 there is no support for dotnet core 2.2 as well.

The error looks like this and it is expected due to the how *targets* file is implemented.
```
 Orleans-CodeGen - Generating file /home/rakkattakka/Development/reviewiteasy/ReviewItEasy.Webhooks/src/ReviewItEasy.Webhooks.Common/obj/Debug/netcoreapp2.2/ReviewItEasy.Webhooks.Common.orleans.g.cs
  -- Code Generation FAILED -- 
  
  Exc level 0: System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
  
     at System.ModuleHandle.ResolveType(RuntimeModule module, Int32 typeToken, IntPtr* typeInstArgs, Int32 typeInstCount, IntPtr* methodInstArgs, Int32 methodInstCount, ObjectHandleOnStack type)
     at System.ModuleHandle.ResolveTypeHandleInternal(RuntimeModule module, Int32 typeToken, RuntimeTypeHandle[] typeInstantiationContext, RuntimeTypeHandle[] methodInstantiationContext)
     at System.Reflection.RuntimeModule.ResolveType(Int32 metadataToken, Type[] genericTypeArguments, Type[] genericMethodArguments)
     at System.Reflection.CustomAttribute.FilterCustomAttributeRecord(CustomAttributeRecord caRecord, MetadataImport scope, Assembly& lastAptcaOkAssembly, RuntimeModule decoratedModule, MetadataToken decoratedToken, RuntimeType attributeFilterType, Boolean mustBeInheritable, Object[] attributes, IList derivedAttributes, RuntimeType& attributeType, IRuntimeMethodInfo& ctor, Boolean& ctorHasParameters, Boolean& isVarArg)
     at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeModule decoratedModule, Int32 decoratedMetadataToken, Int32 pcaCount, RuntimeType attributeFilterType, Boolean mustBeInheritable, IList derivedAttributes, Boolean isDecoratedTargetSecurityTransparent)
     at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType)
     at System.Attribute.GetCustomAttributes(Assembly element, Type attributeType, Boolean inherit)
     at System.Reflection.CustomAttributeExtensions.GetCustomAttributes[T](Assembly element)
     at Orleans.ApplicationParts.AssemblyAttributeFeatureProvider`1.PopulateFeature(IEnumerable`1 parts, TFeature feature)
     at Orleans.ApplicationParts.ApplicationPartManager.PopulateFeature[TFeature](TFeature feature)
     at Orleans.ApplicationPartManagerExtensions.CreateAndPopulateFeature[TFeature](IApplicationPartManager applicationPartManager)
     at Orleans.CodeGenerator.RoslynCodeGenerator..ctor(IApplicationPartManager partManager, ILoggerFactory loggerFactory)
     at Orleans.CodeGeneration.CodeGenerator.GenerateSourceForAssembly(Assembly grainAssembly, LogLevel logLevel)
     at Orleans.CodeGeneration.CodeGenerator.GenerateCodeInternal(CodeGenOptions options)
     at Orleans.CodeGeneration.CodeGenerator.GenerateCode(CodeGenOptions options)
     at Orleans.CodeGeneration.Program.Main(String[] args)
/home/rakkattakka/.nuget/packages/microsoft.orleans.orleanscodegenerator.build/2.1.2/build/Microsoft.Orleans.OrleansCodeGenerator.Build.targets(82,5): error MSB3073: The command ""/usr/share/dotnet/dotnet" "/home/rakkattakka/.nuget/packages/microsoft.orleans.orleanscodegenerator.build/2.1.2/build/../tasks/netcoreapp2.0/Orleans.CodeGeneration.Build.dll" "@obj/Debug/netcoreapp2.2/codegen/ReviewItEasy.Webhooks.Common.orleans.g.args.txt"" exited with code 3. [/home/rakkattakka/Development/reviewiteasy/ReviewItEasy.Webhooks/src/ReviewItEasy.Webhooks.Common/ReviewItEasy.Webhooks.Common.csproj]
```

I'm wondering whether there are plans to support netcore 2.2 with the release?

Thanks.

--- a/README.md
+++ b/README.md
@@ -24,12 +24,12 @@ There are several packages, one for each different project type (interfaces, gra
 In the grain interfaces project:
 ```
 PM> Install-Package Microsoft.Orleans.Core.Abstractions
-PM> Install-Package Microsoft.Orleans.OrleansCodeGenerator.Build
+PM> Install-Package Microsoft.Orleans.CodeGenerator.MSBuild
 ```
 In the grain implementations project:
 ```
 PM> Install-Package Microsoft.Orleans.Core.Abstractions
-PM> Install-Package Microsoft.Orleans.OrleansCodeGenerator.Build
+PM> Install-Package Microsoft.Orleans.CodeGenerator.MSBuild
 ```
 In the server (silo) project:
 ```

